Tmain/list-params.d/stdout-expected.txt

```diff
@@ -65,4 +65,3 @@ define	define replacement for an identifier (name(params,...)=definition)
 if0	examine code within "#if 0" branch (true or [false])
 ignore	a token to be specially handled
 pragma	handle program
-
```

Comment:
About this file, the last newline should be kept.

Reply:
Fixed